views/layouts/application.ejs

```diff
@@ -43,6 +44,9 @@
                     <li>
                         <a href="/#faq">FAQ</a>
                     </li>
+                    <li>
+                        <a href="/privacy">Privacy</a>
```

Comment:
I wouldn't put this so prominently in the title, can you find a place in the footer maybe?

Reply:
No problem, will do - I need to address a styling issue with the footer either way.